.github/workflows/visual-regression.yml

```
@@ -0,0 +1,58 @@
+name: Visual regression with Netlify
```

I think this is a kind of regression tests to detect the degradation of visualization functions.
So, how about adding `tests` to the name?
```suggestion
name: Visual regression tests with Netlify
```

---

Agreed. I revised the code.

---

Thank you.